data/brands/shop/mobile_phone.json

```diff
@@ -10,15 +10,15 @@
   },
   "items": [
     {
-      "displayName": "3 Store",
+      "displayName": "",
```

Comment:
It needs a `displayName` here.  It should probably be the same as the name "Three."

Reply:
Yeah, I forgot to replace the displayName